src/decision/dMCDA.jl

```diff
@@ -784,3 +783,37 @@ function unguided_site_selection(prefseedsites, prefshadesites, seed_years, shad
 
     return prefseedsites[prefseedsites.>0], prefshadesites[prefshadesites.>0]
 end
+
+
+"""
+    mean_var_offset(env_layer::NamedDimsArray dims::Union{Symbol,Tuple{Symbol,Symbol}}; w=0.5)
+Calculates mean over specified dimensions plus half the standard deviation.
+
+# Arguments
+- `env_layer` : Environmental data layer to calculate the mean of.
+- `dims` : Dimensions to aggregate over.
+- `w` : Weighting for std offset to mean.
+
+# Returns
+- Mean + std of input env_layer, where std is weighted by (1-w) and aggregated over dims.
+"""
+function mean_var_offset(env_layer::NamedDimsArray dims::Union{Symbol,Tuple{Symbol,Symbol}}; w=0.5)
```

Comment:
The function is not intended to calculate an offset

https://github.com/open-AIMS/ADRIA.jl/pull/434#discussion_r1308503109